### DISCUSSION TITLE
Deep Dive: Astroid Boilerplate Site 205

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#799] [Video Tutorials]
Hi, hi

back from holidays and am once again making Astroid Tutorials. I hope it is ok when I post them here as I make them?

https://astroidframe.work/blog/astroid-hands-on/deep-dive-astroid-boilerplate-site-205

Aiming to make 1-2 of these a month. Please like subscribe, share and help spread the news!

all the best

Paul

**Comment by sonvnn** [comment]
It's amazing @paurray . I can't wait to see a video for Article Layout 😸

**Comment by Chacapamac** [comment]
Ok Mr. @paurray  GREAT VIDEO!
So clear and useful. Bravo!

**Comment by paurray** [comment]
@sonvnn @Chacapamac 
Thanks for tor the ongoing support. I am doing a lot of experimenting with different formats. And am also curious where this goes.

I will post new content here as it comes out. Stay tuned!

> Article Layout

*Noted
